examples/jenkins-integration/jenkinsfile

```diff
@@ -0,0 +1,175 @@
+pipeline {
+    agent any
+
+    environment {
+        def provision_successful = false
+        def vdb_id = ""
+        def vdb_name = ""
+        def DCT_HOSTNAME = "<INSERT>"
+        def SOURCE_VDB = "<INSERT>"
+        test_error = 'false'
+    }
+
+    stages {
+
+        stage ('Build Application') {
+            steps {
+                echo ('Building...')
+                sleep (50)
+            }
+        }
+        
+        stage ('Stand Up Full Application') {
+            parallel {
+                stage ('Apply Application Install') {
+                    steps{
+                        echo ('Provisioning Test App...')
+                        sleep (30)
+                    }
+                }
+                stage ('Create Database w/ Terraform') {
+                    steps {
+                        script {
+                            echo ('Provisioning  VDB...')
+                            git branch: 'main', changelog: false, poll: false, url: 'https://github.com/nick-mathison/delphix-terraform-examples.git'
+                            // sh ('ls -R')
+                            sh ('terraform -chdir=simple-provision init')
+                            withCredentials([string(credentialsId: 'DCT_API_KEY', variable: 'KEY')]) {
+                                sh ('terraform -chdir=simple-provision apply -var="source_data_id_1=$SOURCE_VDB" -var="dct_hostname=$DCT_HOSTNAME" -var="dct_api_key=$KEY" -auto-approve')
+                            }
+                            vdb_id = sh(script: 'terraform -chdir=simple-provision output vdb_id_1', returnStdout: true)
+                            vdb_id = vdb_id.replaceAll('\\"', "").trim()
+                            vdb_name = sh(script: 'terraform -chdir=simple-provision output vdb_name_1', returnStdout: true)
```

Comment:
Update chdir value to if you bring in the full folder

`examples/simple-provision`

Reply:
done